ramls/staging_user.json

```diff
@@ -5,15 +5,13 @@
   "type": "object",
   "properties": {
     "isEmailVerified": {
-      "description": "A flag to determine if a patron is activated/email verification completed.",
-      "type": "boolean",
-      "default": false
+      "description": "A boolean flag that indicates whether the patron has completed email verification. If this value is not provided when creating a new record, it will default to false.",
```

Comment:
I think we should also specify this "The value is required to be true for the Kiosk case of user registration since the email verification step is skipped in that scenario."

Reply:
A boolean flag indicates if the patron has completed email verification. By default, this value is set to false when creating a new record. However, for Kiosk user registrations, this value should be sent true.

Maybe something like this